designs/administrator/4.3/users-groups/index.md

```diff
@@ -0,0 +1,34 @@
+---
+parent: Administrator
```

Comment:
@andybraren Is there still any need to edit the admin/version number indexes to include this design? Or does just including these accomplish that? Thanks.

Reply:
These two fields are all that’s needed 👍

Reply:
Awesome, very easy thanks!